docs/installation/local.md

```diff
@@ -68,27 +68,7 @@ CREATE DATABASE oevent WITH OWNER john;
 * Once database is created, exit the psql shell with `\q` followed by ENTER.
 
 
-* **Step 3** - Install bower and frontend requirements. For this you need to be present in the root directory of the project. The root directory contains the file ```bower.json```. When you write ```bower install```, it finds bower.json and installs the libraries on the system. Bower has been removed as a dependency in the nextgen version of the project. Please skip this step if working on nextgen branch.
-
-```sh
-npm install bower -g
-bower install
-```
-
-if error
-```sh
-sudo npm install bower -g
-sudo ln -s /usr/bin/nodejs /usr/bin/node
-sudo bower install
-```
-
-for mac user:
-```sh
-sudo npm install bower -g
-bower install
-```
-
-* **Step 4** - Create application environment variables.
+* **Step 3** - Create application environment variables.
 
 ```sh
 cp .env.example .env
```

Comment:
@tulikavijay I guess you forgot to remove the command `cp .env.example .env` which would be unnecessary if bower is not a requirement.

Reply:
Also remove the `Note` at the end because that again won't be necessary after removing bower as a requirement.
Also squash your commits.

Reply:
@tulikavijay .env file has no direct relation with bower, please put the instruction back :-)